raft/progress.go

```diff
@@ -64,12 +64,17 @@ type Progress struct {
 	RecentActive bool
 
 	// inflights is a sliding window for the inflight messages.
+	// Each inflight message might contain more than one entries
+	// and has a size limit defined in raft config as MaxSizePerMsg.
```

Comment:
A size limit in number of bytes or number of entries? I'm again confused about the purpose of this whole thing. Are the number of bytes tracked anywhere?